src/main/java/org/mifos/connector/slcb/utils/SLCBUtils.java

```diff
@@ -77,4 +78,18 @@ public static Payee convertTransactionToPayee(Transaction transaction) {
         return payee;
     }
 
+    public static int getStatusCodeOrDefault(Status status) {
+        if (status != null) {
+            return status.getCode();
+        }
+        return -1;
```

Comment:
use an invalid status enum for better readability,  something like `Status.INVALID.getCode()`

Reply:
noted

Reply:
@fynmanoj done, please recheck